lib/tasks/check_all.rake

```diff
@@ -0,0 +1,13 @@
+task check_all: :environment do
+  run('bundle exec brakeman . -z -q', expected_output: 'No warnings found')
+  run('bundle exec rubocop -a app config lib spec', expected_output: 'no offenses detected')
+  run('bundle exec reek app config lib spec', expected_output: '0 total warnings')
+  run('bundle exec rails_best_practices .', expected_output: 'No warning found. Cool!')
+  system('rspec')
+end
+
+def run(command, expected_output:)
+  `#{command}`.tap do |output|
+    puts output unless output.include?(expected_output)
+  end
+end
```

Comment:
That is to avoid the intermediate output of all validations but rspec. Each one of these validations writes an output in case of success. I wanted to mimic the reporting of rspec where it gives details only on the failed cases, and if nothing is logged then the tests it ok.

The reason that rspec runs at the end is that some of these lints have some flags to modify the source code. Even if those flags should be used in the script and the modifications should perform only refactors and no changes in the logic I still wanted rspec to run on the final version of the files.

Reply:
Mmm.. I got your point, but I still not sure if this should be on the repository, maybe you can configure your local git by adding a `pre_push` or `pre_commit` to check those things.
Here is an example: https://www.git-scm.com/docs/githooks#_pre_push
Here a documentation: https://gist.github.com/gmodarelli/5b92589ffbe5478628cf
What do you think @santiagovidal ?